api/package.json

```diff
@@ -22,15 +22,16 @@
     "node": "^16.13"
   },
   "dependencies": {
-    "@googlemaps/google-maps-services-js": "^3.1.16",
-    "@nestjs/common": "^7.6.15",
-    "@nestjs/core": "^7.6.15",
-    "@nestjs/jwt": "^7.2.0",
-    "@nestjs/passport": "^7.1.5",
-    "@nestjs/platform-express": "^7.6.15",
-    "@nestjs/swagger": "^4.8.0",
-    "@nestjs/terminus": "^7.2.0",
-    "@nestjs/typeorm": "^7.1.5",
+    "@googlemaps/google-maps-services-js": "^3.3.2",
+    "@nestjs/axios": "^0.0.3",
+    "@nestjs/common": "^8.2.1",
+    "@nestjs/core": "^8.2.1",
+    "@nestjs/jwt": "^8.0.0",
+    "@nestjs/passport": "^8.0.1",
+    "@nestjs/platform-express": "^8.2.1",
+    "@nestjs/swagger": "^5.1.4",
+    "@nestjs/terminus": "^8.0.1",
+    "@nestjs/typeorm": "^8.0.2",
```

Comment:
I know this is the package you enjoyed the most updating :)
But shouldn't we aim to pin dependencies (at least the ones that are not development dependencies) ? 
Or there is a reason to not do so?

Reply:
I guess it makes more sense to change it to ~ so we can have bugfix versions updates. I'll take care of it